www/docs/configuration/callbacks.md

```diff
@@ -170,7 +170,7 @@ callbacks: {
   jwt: async (token, user, account, profile, isNewUser) => {
     const isSignIn = (user) ? true : false
     // Add auth_time to token on signin in
-    if (isSignIn) { token.auth_time = new Date().toISOString() }
+    if (isSignIn) { token.auth_time = Date.now() }
```

Comment:
@iaincollins `Date.now()` returns a timestamp in milliseconds while the claim is defined as seconds since epoch. If you're relying on `jose` to validate the auth_time for you and pass in milliseconds you're effectively never going to hit a point when the claim validation fails.

Reply:
Oh good catch!

We don't use `auth_time` internally (we use `iat` / `exp`), this is just an example that shows folks how to use the callback in case they need to (e.g. for interoperability) but thank you - I'll update the example!